### ISSUE TITLE
Bug: Color differentiation of the code syntax for people who has color vision deficiency

### ISSUE THREAD

**Issue by applegz** [mdn/yari#5423]
# Summary
The color in code snippets might create difficulties for people who are colorblind to tell the difference, especially in the dark mode.

## Where did you spot the bug?
I first notice on this [page](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String) in dark mode, I believe it's the same on every page that has code snippets.

## What is the problem?
Current code snippet color with regular rendering in dark mode:
![Screen Shot 2022-03-02 at 12 04 37 PM](https://user-images.githubusercontent.com/18157459/156411469-59e6e673-7e50-4aec-91b1-98a3accfc3a8.png)
Current code snippet color in emulator with deuteranopia vision deficiency in dark mode:
![Screen Shot 2022-03-02 at 12 04 51 PM](https://user-images.githubusercontent.com/18157459/156412166-ffc86dcd-1001-4e1f-8ddd-4dbcd13ecfe5.png)

It's also an issue not in the dark mode:
![Screen Shot 2022-03-02 at 12 12 11 PM](https://user-images.githubusercontent.com/18157459/156412730-a94e391a-c4d4-4faf-857b-321503b0ae18.png)
![Screen Shot 2022-03-02 at 12 12 23 PM](https://user-images.githubusercontent.com/18157459/156412739-52bbe377-020f-424f-8c10-eea930a62737.png)

## What did you expect to happen?
Everyone should be able to tell the code syntax by just looking at the color. It should be accessible for anybody.

## Steps to reproduce the issue

1. Open any Mozilla website that has code snippet in Chrome
2. Open the console
3. Click ... to open more menu
4. Click `more tools`
5. Select `rendering`
6. Find `emulate vision deficiencies`
7. Select different options to see what people might see if they have color vision deficiency

### Device

- [x] Desktop
- [ ] Mobile: ...
- [ ] Tablet: ...
- [ ] Other: ...

### Browser

- [x] Chrome: (what version?) 98
- [ ] Firefox: (what version?)
- [ ] Safari: (what version?)
- [ ] Other: (what browser / version?)

### Operating system

- [ ] Windows: (what version?)
- [x] Mac OS: (what version?) 12.2.1
- [ ] Linux: (what distribution / version?)

**Comment by caugner** [comment]
@applegz Can you please check [the example](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String#creating_strings) again in dark mode? It looks to me like the issue is resolved.

**Comment by caugner** [comment]
This appears to be resolved meanwhile, especially in Dark mode, whereas in Light mode the difference is more subtle:

| Rendering   | Dark | Light |
|-------------|------|-------|
| (default)   | <img width="780" height="322" alt="Image" src="https://github.com/user-attachments/assets/e6fa50e6-86d9-42d3-b099-17dc861098f0" /> | <img width="780" height="322" alt="Image" src="https://github.com/user-attachments/assets/986e99c5-6755-45ed-b676-73845957235b" /> |
| Deuteranopia |<img width="780" height="322" alt="Image" src="https://github.com/user-attachments/assets/b9070149-45fa-4b4a-a454-e0ea65fcc4f2" /> | <img width="780" height="322" alt="Image" src="https://github.com/user-attachments/assets/8c4150f0-fd2e-4d50-b2aa-cc2d7392e312" /> |
